### PR TITLE
Scatter spit Range, Scatter, Damage

### PR DESCRIPTION
**Pull request by vilereaver41**
## About The Pull Request

Changes scatter spit  scatter rate and damage

## Why It's Good For The Game

Scatter spit isn't very good right now, it does little damage even if you manage to PB people. right now its a mosquito bite. and when you get in the range you are oftentimes punished by losing half your hp to marine fire just to use the ability. this at least makes it more usable. 

this should help make Spitter more useful as its spit damage even at ancient does little damage that is including the 15%+ per maturity.  

Abilities like this are overlooked and their damage is considered "ok" but people don't take into account acid armor, not to mention the projectile's speed is quite slow compared to marine bullets.

the ability will now be useful.

## Changelog
:cl:
balance: scatter is now 2
balance: projectile damage is now 20 from 10.
/:cl:

### CODE REVIEW

**Comment by KrabSpider** [comment]
Hi. It is literally the equivalent of 2 spits. That's 18 damage after medium armor calcs. There is hardly any point in trying to put scatterspit off if
1) Channeling it means you are _going_ to be shot away as you prep it.
2) It is much more costly than simply spitting at a marine

**Comment by maikilangiolo** [comment]
Scatter spit already has greater range than view screen and bypasses wired cades.

**Comment by David-Stormwell** [comment]
Damage is an extreme jump, nothing more then x2 should be done.

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by vilereaver41** [comment]
> Damage is an extreme jump, nothing more then x2 should be done.

done made it 20

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -1902,12 +1902,12 @@ datum/ammo/bullet/revolver/tp44
...
 /datum/ammo/xeno/acid/heavy/scatter
-	damage = 10
+	damage = 25
 	flags_ammo_behavior = AMMO_XENO|AMMO_EXPLOSIVE|AMMO_SKIPS_ALIENS
 	bonus_projectiles_type = /datum/ammo/xeno/acid/heavy/scatter
 	bonus_projectiles_amount = 6
-	bonus_projectiles_scatter = 3
-	max_range = 8
+	bonus_projectiles_scatter = 2
+	max_range = 10
```
[the remark this answers is not in the document]
Comment (vilereaver41) [reply code/modules/projectiles/ammo_datums.dm]: done

**Review by BraveMole — APPROVED** [review]

**Pull request merged**